pinot-core/src/main/java/com/linkedin/pinot/core/io/readerwriter/impl/FixedByteSingleColumnSingleValueReaderWriter.java

```diff
@@ -108,18 +110,21 @@ public FixedByteSingleValueMultiColReader getReader() {
   /**
    * @param numRowsPerChunk Number of rows to pack in one chunk before a new chunk is created.
    * @param columnSizesInBytes
-   * @param description
+   * @param memoryManager
+   * @param columnName
    */
-  public FixedByteSingleColumnSingleValueReaderWriter(int numRowsPerChunk, int columnSizesInBytes, String description) {
+  public FixedByteSingleColumnSingleValueReaderWriter(int numRowsPerChunk, int columnSizesInBytes,
+      OffHeapMemoryManager memoryManager, String columnName) {
     chunkSizeInBytes = numRowsPerChunk * columnSizesInBytes;
     this.numRowsPerChunk = numRowsPerChunk;
     this.columnSizesInBytes = columnSizesInBytes;
-    this.description = description;
+    this.memoryManager = memoryManager;
```

Comment:
The creator of memory manager is expected to close it (in our case there is only one creator, RealtimeSegmentImpl)